app.py

```diff
@@ -3,17 +3,26 @@
 import logging
 import requests
 import openai
-from flask import Flask, Response, request, jsonify
+from flask import Flask, Response, request, jsonify, send_from_directory
 from dotenv import load_dotenv
 
 load_dotenv()
 
-app = Flask(__name__)
+app = Flask(__name__, static_folder="static")
+
+# Static Files
```

Comment:
This looks reasonable, assuming it works.

Related Flask docs:
https://flask.palletsprojects.com/en/2.3.x/patterns/singlepageapplications/
https://flask.palletsprojects.com/en/2.3.x/api/#flask.send_from_directory
https://flask.palletsprojects.com/en/2.3.x/api/#flask.Flask.send_static_file